sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/AnswerCallOptions.java

```diff
@@ -100,4 +114,15 @@ public AnswerCallOptions setAzureCognitiveServicesUrl(String azureCognitiveServi
         this.azureCognitiveServicesUrl = azureCognitiveServicesUrl;
         return this;
     }
+    
+     /**
+     * Set the operationContext.
+     *
+     * @param operationContext the operationContext to set
+     * @return the AddParticipantOptions object itself.
```

Comment:
AddParticipantOptions -> answercalloptions

Reply:
Good catch, thank you!

Reply:
fixed